src/redux/books/books.js

```diff
@@ -1,26 +1,93 @@
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
-
+const apiUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/Gqq4TbGrZmbMnwiy8Tsp/books';
+const GET_BOOKS = 'bookstore/books/GET_BOOKS';
 const initialState = [];
 
-export const addbook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
+const getBooksFromApi = async () => {
+  const response = await fetch(apiUrl, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
```

Comment:
- [ ] It is a requirement that you use `createAsyncThunk` in the project for creating thunk action creators. Kindly do so for all.
Requirements Screenshot
![image](https://user-images.githubusercontent.com/81932478/202865719-09b1adc1-4677-46b1-96f3-6c91d6e5d848.png)